doc/classes/RichTextLabel.xml

```diff
@@ -409,6 +409,7 @@
 			<param index="1" name="base_direction" type="int" enum="Control.TextDirection" default="0" />
 			<param index="2" name="language" type="String" default="&quot;&quot;" />
 			<param index="3" name="st_parser" type="int" enum="TextServer.StructuredTextParser" default="0" />
+			<param index="4" name="jst_flags" type="int" enum="TextServer.JustificationFlag" default="163" />
```

Comment:
The `jst` abbreviation may be a bit awkward, it might be better to go with the full `justification_flags`.